src/pages/RepoPage/FlagsTab/subroute/FlagsTable/hooks/useRepoFlagsTable.ts

```diff
@@ -1,23 +1,28 @@
 import { format, sub } from 'date-fns'
-import { useCallback, useState } from 'react'
 import { useParams } from 'react-router-dom'
 
-import { SortingDirection } from 'old_ui/Table/constants'
 import {
   AFTER_DATE_FORMAT_OPTIONS,
   MEASUREMENT_TIME_INTERVALS,
+  TIME_OPTION_KEY,
 } from 'pages/RepoPage/FlagsTab/constants'
 import { useLocationParams } from 'services/navigation'
 import { useRepo } from 'services/repo'
 import { useRepoFlags } from 'services/repo/useRepoFlags'
 
-const getSortByDirection = (isDesc) =>
-  isDesc ? SortingDirection.DESC : SortingDirection.ASC
+const getSortByDirection = (isDesc: boolean) => (isDesc ? 'DESC' : 'ASC')
 
-const createMeasurementVariables = (historicalTrend, oldestCommitAt) => {
-  const isAllTime = !Boolean(historicalTrend) || historicalTrend === 'ALL_TIME'
+const createMeasurementVariables = (
+  historicalTrend: TIME_OPTION_KEY,
+  oldestCommitAt?: string | null
+) => {
+  const isAllTime = historicalTrend === 'ALL_TIME'
+  const getOldestCommitAt = () =>
+    oldestCommitAt
+      ? new Date(oldestCommitAt)
+      : sub(new Date(), { ...AFTER_DATE_FORMAT_OPTIONS.LAST_6_MONTHS })
```

Comment:
thoughts on moving this from a nested ternary to just the default at the function header?

Reply:
You are smart @ajay-sentry 🥔